### ISSUE TITLE
Force Bundling on Map

### ISSUE THREAD

**Issue by WPSlacker** [carnby/matta#2]
I tried to find an example of matta with Force Bundling on a map, such as seen here, 'map of Santiago':
http://carnby.github.io/

Can you please provide an ipython notebook example of this beautiful visualization?

**Comment by carnby** [comment]
Ok! I will work on that. 

Meanwhile, you can see an example of a scaffolded visualization with force bundling [here](http://dcc.uchile.cl/~egraells/abrecl/#sec-comunas).

**Comment by WPSlacker** [comment]
Yes, your examples are amazing!  I am just having problems with the workflow process and placing the D3js network on the map (leaflet, I think?) using matta.
Thanks!

**Comment by carnby** [comment]
Probably this weekend I will upload a new version of matta, including a notebook with a force bundling example:

![Example](https://pbs.twimg.com/media/B_wzUncUwAAXiU3.png:large)

Do you have any special doubt about the workflow, so I can put emphasis on that in the example?

**Comment by WPSlacker** [comment]
Wow, that's amazing.  I've been trying to use a large data set, approximately 1 million records.  I condensed it into an incidence matrix of simple counts between locations.  But, neither using the entire dataset, nor the condensed dataset worked.

So, I guess my problem is that of getting the correct line density, given the count for the number of lines between two particular locations.

I hope that makes sense.
